sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ClientSideRequestStatistics.java

```diff
@@ -99,6 +101,15 @@ public void recordResponse(RxDocumentServiceRequest request, StoreResultDiagnost
         storeResponseStatistics.storeResult = storeResultDiagnostics;
         storeResponseStatistics.requestOperationType = request.getOperationType();
         storeResponseStatistics.requestResourceType = request.getResourceType();
+        if (request.requestContext.sessionToken == null && request.getOriginalSessionToken() == null) {
+            storeResponseStatistics.requestSessionToken = null;
+        } else {
+            if (request.getOriginalSessionToken() != null) {
+                storeResponseStatistics.requestSessionToken = request.getOriginalSessionToken();
+            } else {
+                storeResponseStatistics.requestSessionToken = SessionTokenHelper.concatPartitionKeyRangeIdWithSessionToken(request.requestContext.resolvedPartitionKeyRange.getId(), request.requestContext.sessionToken.convertToString());
```

Comment:
consider break the line for readability(can start trying with 120 characters/line)